docs/user/guides/creating-config-file.rst

```diff
@@ -0,0 +1,100 @@
+================================
+Creating your configuration file
+================================
+
+As part of the initial setup for your Read the Docs site, you need to create a 
+**configuration file** called ``.readthedocs.yaml``.
+
+The configuration file like a personalized guidebook that tells the platform 
+what specific settings for your project. Every project is unique, and by using 
+a configuration file, you can tailor the behavior of Read the Docs to match 
+your specific needs.
+
+This how-to guide covers
+#. Where to put your configuration file
+#. The required parts of your configuration file
+
+This should be enough to get you started!
+
+.. tip::
+   
+   The complete list of all possible ``.readthedocs.yaml`` settings, including 
+   the optional settings not covered in on this page, is found on 
+   :doc:`the configuration file reference page </docs/user/config-file/index.rst>`.
+
+####################################
+Where to put your configuration file
+####################################
+
+The ``.readthedocs.yaml`` file should be placed in the **root directory**. The 
+root directory is the top-most location of your repository.
+
+To make the file, navigate to your root directory and create a new 
+file with the name ``.readthedocs.yaml``.
+
+*************************
+The config file is hidden
+*************************
+
+The ``.`` at the beginning of the file indicates that the file is hidden. If 
+the file browser you are using is not configured to show hidden files, you may 
+not be able to see the file after you create it!
+
+If you cannot see ``.readthedocs.yaml`` after creating it, use your favorite 
+search engine to find instructions on showing hidden files for your specific 
+operating system.
+
+#############################################
+The required parts of your configuration file
+#############################################
```

Comment:
It would be great if the how-to showed some minimal copy-paste friendly examples in a tabbed interface.

I think it's important to really trim out all explanation. It could be enough to explain in one paragraph what an example file does, but users should lookup configuration values in the reference to know more.

For a start, tabs could be: Sphinx, MkDocs, Custom builder

I agree with @humitos's [comment](https://github.com/readthedocs/readthedocs.org/pull/10301#issuecomment-1544576764) that it will be useful to have even more templates. Where and how to maintain them over time is important, but I think we might as well handle that separately (New issue: "Factor out .readthedocs.yaml examples from documentation and test that they work in real builds" - and guess what? we now support monorepo structures with multiple .readthedocs.yaml files, so we can store them in different folders in our docs and setup a live RTD project for each of them!)

```
tabs::

    tab:: Sphinx

        Here is a tab with a Sphinx config file example

    tab:: MkDocs

        Here is a tab with a Sphinx config file example

    tab:: Custom builder

        Example using build.commands

```

Reply:
We talked about this for our configuration file deprecation as well. No need to hold this PR up with that though, but I think we will end up with some example here as we start to push users towards the configuration file more.

Reply:
Would you also think that these 3 examples are the most important ones?

I like the approach of having some commented-out configuration options showcased in the examples, too (as the new Import Wizard step does)

Reply:
Sphinx/Mkdocs are obvious, but we should cover specific doc tools instead of a generic "Custom builder". Again though, not a priority for this initial pass. There is a lot of information we need to cover on this page.

I'm not even sure this is the page we should cover these examples on. The example for each tool is more reference than anything. I would still suggest punting on this concept.

Reply:
Sounds good. We can use the tab to get people started with another tool and give them the pointers they need to continue using the reference.

Reply:
The how-to now has a structure where the user is encouraged to start with one of the templates selected in a tabbed interface.

Examples are maintained in separate files that can be reused across our docs and platform.